rulesetCS.xml

```diff
@@ -2,6 +2,17 @@
 <ruleset name="ruleset">
     <description>ShopSys Coding Standards</description>
 
+    <!-- From php-md -->
```

Comment:
By the way: What is the rationale behind moving checks from PHPMD to PHP-CS-Fixer?

Reply:
Oh, I can see now: https://github.com/shopsys/coding-standards/issues/7
You want to completely replace PHPMD.

Reply:
One more thing: I do not see any reason to put the fixers that were PHPMD rules before in separate section called `<!-- From php-md -->`. For anybody in future it is irrelevant where were the rules before.

It is good for a reviewer like me to realize what happened but this is the kind of information that belongs into commit message.

Can you please merge the items from `<!-- From php-md -->` int to the `<!-- Custom ShopSys standards -->` section? Also note that the rules are kept in alphabetical order (for better clarity).

Reply:
You are right. I agree!